CondTools/Geometry/plugins/DTRecoIdealDBLoader.cc

```diff
@@ -30,16 +43,22 @@ void DTRecoIdealDBLoader::beginRun(const edm::Run&, edm::EventSetup const& es) {
     return;
   }
 
-  edm::ESTransientHandle<DDCompactView> pDD;
   edm::ESHandle<MuonGeometryConstants> pMNDC;
-  es.get<IdealGeometryRecord>().get(pDD);
-  es.get<IdealGeometryRecord>().get(pMNDC);
-
-  const DDCompactView& cpv = *pDD;
   DTGeometryParsFromDD dtgp;
 
-  dtgp.build(&cpv, *pMNDC, *rig);
-
+  if (fromDD4Hep_) {
+    edm::ESTransientHandle<cms::DDCompactView> pDD;
+    es.get<IdealGeometryRecord>().get(pDD);
```

Comment:
`es.get()` is deprecated. If you can, it would be good to change this code to the recommended way:
https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideHowToGetDataFromES

Reply:
@cvuosalo : yes but, before perform this change, I'd prefer to commit the other modifications, I made for the dd4hep::mm, that you requested then, if the tests and histos comparison will be ok I'll proceed with this point.

Reply:
Yes, the EventsSetup changes can be made in a later PR if you wish.

Reply:
@cvuosalo : I'll try to do it in this PR after the test will be finished